about/index.md

```diff
@@ -4,8 +4,63 @@
 ## Banner code and credit
 This [banner](https://raw.githubusercontent.com/pymc-devs/pymc-data-umbrella/main/banner.png) is generated from [this code](https://raw.githubusercontent.com/pymc-devs/pymc-data-umbrella/main/banner.py), the code in this link is a trivial customization of the [original code](https://github.com/pymc-devs/pymcon/blob/gh-pages/assets/make_trajectories.py) by [Colin Caroll](https://colindcarroll.com) who designed a [similar banner for pymcon'20](https://pymcon.com/), Colin is amazing at visualization stuff and even has a couple of [talks](https://github.com/ColCarroll/yourplotlib) about it!!
 
+## Organizers
+
+::::{grid} 2 2 4 4
+:gutter: 3
+
+:::{grid-item-card}
+:link: https://mjhajharia.com
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Meenal Jhajharia" class="rounded-circle" src="../_static/meenal.jpg">
+
+Meenal  Jhajharia
+:::
+:::{grid-item-card}
+:link: https://reshamas.github.io
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Reshama Shaikh" src="../_static/reshama.jpg" class="rounded-circle">
+
+Reshama Shaikh
+:::
+:::{grid-item-card}
+:link: https://mjhajharia.com
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Beryl Kanali" src="../_static/beryl.JPG" class="rounded-circle">
+
+Beryl Kanali
+:::
+:::{grid-item-card}
+:link: https://oriolabril.github.io/oriol_unraveled/
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Oriol Abril Pla" src="../_static/oriol.jpg" class="rounded-circle">
+
+Oriol Abril Pla
+:::
+::::
+
+Additionally, we thank [Ravin Kumar](https://ravinkumar.com)(Volunteer), [Austin Rochford](https://austinrochford.com)(Speaker), [Ricardo Vieira(](https://github.com/ricardoV94/)Speaker), [Sayam Kumar](https://github.com/Sayam753)(Volunteer) and [Thomas Wiecki](https://twiecki.io)(Volunteer) and [NUMFOCUS](https://numfocus.org) for their support.
+
 ## About PyMC and Data Umbrella
-See [PyMC about page](https://docs.pymc.io/en/latest/about.html) and [Data Umbrella website](https://www.dataumbrella.org/)
 
-```{include} decision_tree.md
+[Data Umbrella](https://www.dataumbrella.org/) is a global community for underrepresented persons in data science. It is a fiscally hosted project of Open Collective, a registered 501(c)(3) non-profit based in California, USA. Data Umbrella: organizes online speaker series on data science and open source, organizes sprints / hackathons, curates resources on inclusive practices. All levels are welcome, beginners and experts.
+
+[PyMC](https://docs.pymc.io/en/latest/about.html) is a probabilistic programming package for Python that allows users to fit Bayesian models using a variety of numerical methods, most notably Markov chain Monte Carlo (MCMC) and variational inference (VI). Its flexibility and extensibility make it applicable to a large suite of problems. PyMC is a non-profit project under NumFOCUS.
+
+```{toctree}
```

Comment:
moved this to a toctree again because we reference this page from many other places and from itself, including it duplicates the manual anchors we have defined and they stop behaving properly.